samza-core/src/main/java/org/apache/samza/config/loaders/PropertiesConfigLoaderFactory.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.samza.config.loaders;
+
+import org.apache.samza.SamzaException;
+import org.apache.samza.config.Config;
+import org.apache.samza.config.ConfigLoader;
+import org.apache.samza.config.ConfigLoaderFactory;
+
+
+/**
+ * ConfigLoaderFactory which initialize {@link PropertiesConfigLoader} with properties file path specified in
+ * job.config.loader.properties.path.
+ */
+public class PropertiesConfigLoaderFactory implements ConfigLoaderFactory {
+  private static final String KEY = "path";
```

Comment:
minor: Can you please use a bit more descriptive name for the variable? (e.g. `PATH_KEY`)

Reply:
Updated